docs/docs/howto/kafka-cluster-migration/sync-schemas-to-cluster.rst

```diff
@@ -0,0 +1,23 @@
+Synchronize Schemas to cluster
+==============================
+
+To synchronize schemas from Klaw to a Schema Registry cluster, follow the below steps, which involve displaying the topics and schema versions in Klaw and creating them on the Schema Registry cluster.
+
+.. note::  This process deletes all the existing schemas for that subject in target environment and creates new ones based on Klaw metadata.
+
+Ex : If version 1 already exists on target cluster, it will be deleted, and new schemas will be created starting with '2' as new version.
+Schemas will be registered in the same ascending order of versions. If there is a compatibility failure, schemas will be force registered based on force register selection.
+
```

Comment:
This looks like it is meant to be all part of the same note?

![image](https://github.com/aiven/klaw-docs/assets/121855584/da72cd6e-ffbe-4aca-a105-91e97001ce28)